src/configure/configure.ts

```diff
@@ -252,6 +265,7 @@ class Orchestrator {
             if (await this.extractAzureResourceFromNode(node)) {
                 this.context['isResourceAlreadySelected'] = true;
                 this.context['resourceId'] = this.inputs.targetResource.resource.id;
+                return this.inputs.targetResource.resource;
```

Comment:
It looks weird to me that the function return a value in case it is a resource node but otherwise returns null if it is a folder node.
We should keep it generic and not return anything in either case. We are anyways setting `this.context['isResourceAlreadySelected'] = true;` which should be good way to check later whether resource is already selected.

Reply:
updated